perception_dataset/rosbag2/rosbag2_to_non_annotated_t4_converter.py

```diff
@@ -327,6 +346,10 @@ def _convert(self) -> None:
         self._connect_sample_in_scene()
         self._connect_sample_data_in_scene(sensor_channel_to_sample_data_token_list)
         self._add_scene_description(self._scene_description)
+        print(f"Total elapsed time: {time.time() - start:.2f} sec")
+
+        if self._with_vehicle_status:
+            self._convert_vehicle_state()
 
         if self._with_vehicle_status:
             self._convert_vehicle_state()
```

Comment:
Duplicates?

```suggestion
        if self._with_vehicle_status:
            self._convert_vehicle_state()
```

Reply:
Thanks. It might be a mistake in merging.

Reply:
https://github.com/tier4/tier4_perception_dataset/pull/177/commits/73ee7b277cb1fd5826d01a101846cb7292af0414